### DISCUSSION TITLE
How to save the markdown content?

### DISCUSSION THREAD

**Discussion by nightire** [glideapps/glide-data-grid#572] [Q&A] [answered]
I don't think editing the markdown cell will trigger the `onCellEdited` callback. To ensure I have not made a stupid mistake, I clone this repo and run [the story](https://glideapps.github.io/glide-data-grid/?path=/story/glide-data-grid-dataeditor-demos--all-cell-kinds) locally, turns out it didn't work as well.

I also tried [this story](https://glideapps.github.io/glide-data-grid/?path=/story/tests-testcases--markdown-edits), weirdly it is not even set `onCellEdited` callback at all.

So I'm confused right now; how the markdown cells are saved if `onCellEdited` doesn't respond to its content changing?

**Comment by jassmith** [comment, marked as answer]
Confirmed, fix will roll out today

**Reply by jassmith** [reply]
Fixed in 5.2.1 which is now available on npm
